code/game/objects/items/devices/taperecorder.dm

```diff
@@ -310,6 +310,20 @@
 	canprint = FALSE
 	addtimer(VARSET_CALLBACK(src, canprint, TRUE), 30 SECONDS)
 
+/obj/item/taperecorder/verb/WipeTapeInRecorder()
+	set name = "Wipe Tape"
+
+	if(!mytape || mytape.unspooled)
+		return
+	if(recording)
+		return
+	if(playing)
+		return
+	else
+		mytape.used_capacity = 0;
+		mytape.storedinfo = new;
+		mytape.timestamp = new;
+		to_chat(usr, "<span class='notice'>You wipe this tape entirely.")
```

Comment:
```suggestion
	mytape.used_capacity = 0;
	mytape.storedinfo = new;
	mytape.timestamp = new;
	to_chat(usr, "<span class='notice'>You wipe this tape entirely.")
```
You already have early returns set up for this, you don't need an else statement for this.